pallets/scheduler/src/lib.rs

```diff
@@ -616,310 +710,57 @@ pub mod pallet {
 }
 
 impl<T: Config> Pallet<T> {
-    /// Migrate storage format from V1 to V4.
```

Comment:
Can you explain the rationale for removing the migration code? Was it for a past migration on polkadot?